### ISSUE TITLE
[Shortcut] Ctrl+Q for quit

### ISSUE THREAD

**Issue by AlexWayfer** [keeweb/keeweb#574]
Hello!

I want a little feature (new shortcut): `Ctrl+Q` for quit from application.

I can quit from any application on my desktop by `Ctrl+Q` (or `Ctrl+Shift+Q` for apps with tabs, like terminal or browser), but not from KeeWeb :(

Console at DevTools is clean.

Thank you!

```
KeeWeb v1.4.1 (df218ae, 2017-03-26)
Environment: electron v1.4.15
User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) KeeWeb/1.4.1 Chrome/53.0.2785.143 Electron/1.4.15 Safari/537.36
```

**Comment by felberj** [comment]
On osx it works when it is unlocked, but it is disabled when it is locked.

**Comment by antelle** [comment]
@felberj you mean the menu item? I just checked, menu is enabled and ⌘Q is working.

**Comment by isbad** [comment]
Just got same issue, for me in locked and unlocked state "Quit (⌘Q)" disabled in menu and doesn't work as hotkey. 
```
Sierra 10.12.4
KeeWeb v1.4.1 (df218ae, 2017-03-26)
Environment: electron v1.2.1
User-Agent: Mozilla/5.0 (Macintosh; Intel Mac OS X 10_12_4) AppleWebKit/537.36 (KHTML, like Gecko) KeeWeb/1.2.0 Chrome/51.0.2704.63 Electron/1.2.1 Safari/537.36
```

https://www.dropbox.com/s/2ynzwdag161l2z9/Screenshot%202017-05-05%2018.24.54.png?dl=0

**Comment by antelle** [comment]
@isbad it's because of missing titles. If you re-download the app, it will work. There was an error with older electron versions. #539

**Comment by AlexWayfer** [comment]
[AUR `keeweb-desktop` package](https://aur.archlinux.org/packages/keeweb-desktop/) (from this repo release binaries) — no Main Menu, `Ctrl+Q` doesn't work.

[AUR `keeweb` package](https://aur.archlinux.org/packages/keeweb/) (building from sources from this repo) — no Main Menu, but `Ctrl+Q` works :man_shrugging:

**Comment by AlexWayfer** [comment]
>[AUR `keeweb-desktop` package](https://aur.archlinux.org/packages/keeweb-desktop/) (from this repo release binaries) — no Main Menu, `Ctrl+Q` doesn't work.

[Installation script of this package (PKGBUILD)](https://aur.archlinux.org/cgit/aur.git/tree/PKGBUILD?h=keeweb-desktop).

**Comment by Lukasdoe** [comment]
So should we just implement a closing action for this specific keypress in the program code or is it a linux individual thing that requires a special function?

**Comment by antelle** [comment]
@Lukasdoe it's a linux-specific thing, it works everywhere else.

**Comment by AlexWayfer** [comment]
> @Lukasdoe it's a linux-specific thing, it works everywhere else.

It doesn't in Windows.

**Comment by antelle** [comment]
On Windows it’s Alr+F4, it doesn’t work? 😳

**Comment by Lukasdoe** [comment]
How about adding a shortcut option for closing the program in the shortcut settings page?

**Comment by antelle** [comment]
No, apps must be closed in the way it’s done in the OS, now how they want.

**Comment by AlexWayfer** [comment]
> On Windows it’s Alt+F4, it doesn’t work? flushed

I think it works, but I just disputed words "it works everywhere else" in issue with "Ctrl+Q for quit" title.

**Comment by Lukasdoe** [comment]
So it's still a linux-specific bug, how can it be fixed?

**Comment by antelle** [comment]
The bot was misconfigured, sorry for a wrong notification.

**Comment by milahu** [comment]
what would vscode do?

vscode binds Ctrl+Q to "Exit" in
[vscode/src/vs/workbench/electron-sandbox/desktop.contribution.ts](https://github.com/microsoft/vscode/blob/493d27bf48037947094e76fa34998eb45fd2381c/src/vs/workbench/electron-sandbox/desktop.contribution.ts#L63)

<details>

```
vscode $ find . -name '*.ts' -or -name '*.json' | grep -v node_modules/ | xargs grep -Hn -w KeyCode.KeyQ
./src/vs/workbench/services/keybinding/common/windowsKeyboardMapper.ts:289:             _registerLetterIfMissing(CharCode.Q, KeyCode.KeyQ);
./src/vs/workbench/browser/parts/editor/editor.contribution.ts:252:registry.registerWorkbenchAction(SyncActionDescriptor.from(NavigateToLastEditLocationAction, { primary: KeyChord(KeyMod.CtrlCmd | KeyCode.KeyK, KeyMod.CtrlCmd | KeyCode.KeyQ) }), 'Go to Last Edit Location');
./src/vs/workbench/electron-sandbox/desktop.contribution.ts:82:         mac: { primary: KeyMod.CtrlCmd | KeyCode.KeyQ },
./src/vs/workbench/electron-sandbox/desktop.contribution.ts:83:         linux: { primary: KeyMod.CtrlCmd | KeyCode.KeyQ }
./src/vs/workbench/contrib/quickaccess/browser/viewQuickAccess.ts:252:          primary: KeyMod.CtrlCmd | KeyCode.KeyQ,
./src/vs/workbench/contrib/quickaccess/browser/viewQuickAccess.ts:253:          mac: { primary: KeyMod.WinCtrl | KeyCode.KeyQ },
./src/vs/base/common/keyCodes.ts:517:           [47, 0, ScanCode.KeyQ, 'KeyQ', KeyCode.KeyQ, 'Q', 81, 'VK_Q', empty, empty],
./src/vs/platform/keybinding/common/usLayoutResolvedKeybinding.ts:136:                  case ScanCode.KeyQ: return KeyCode.KeyQ;
```

</details>

this seems to cause problems on wayland, see https://github.com/NixOS/nixpkgs/pull/195650
